doozer/doozerlib/cli/release_gen_assembly.py

```diff
@@ -42,8 +42,8 @@ def releases_gen_assembly(ctx, name):
 @click.option("--custom", default=False, is_flag=True,
               help="If specified, weaker conformance criteria are applied "
                    "(e.g. a nightly is not required for every arch).")
-@click.option("--prerelease", is_flag=True,
-              help="Prepare the assembly for a prerelease operator advisory. Only valid for preview and candidate assemblies.")
+@click.option("--pre-ga-mode", default="none", type=click.Choice(["prerelease", "advance"], case_sensitive=False),
```

Comment:
Same here, we can leave the default out which will make it None when not provided

Reply:
added in https://github.com/openshift-eng/art-tools/commit/19496fcd809131a42d9794e14d0a57142ada1afd